org.godotengine.Godot.yaml

```
@@ -39,6 +39,7 @@ build-options:
       udev=no
 finish-args:
   - --share=ipc
+  - --socket=wayland
   - --socket=x11
```

I would suggest you use `--socket=fallback-x11` along with `--socket=wayland` instead of using both `--socket=wayland` and `--socket=x11` together. This ensures that the Flatpak app doesn't unnecessarily use X11 while it is using Wayland, while also ensuring it still works under X11.
```suggestion
  - --socket=fallback-x11
```

---

Ordinarily, that's the right thing to do, but as I stated in my commit message it would be incorrect here:

> --socket=x11 is left as-is rather than being replaced with fallback-x11
> because X11 remains the default; manual intervention is needed to
> configure the editor and/or game to use Wayland.

With your proposed change the editor would fail to start on Wayland sessions.

---

Ah. I missed it somehow. Sorry.

If that's the case, then we can leave the `finish-args` as they are. Looks like this PR should be good to merge :+1: